ghw/libghw.c

```diff
@@ -1149,7 +1247,7 @@ ghw_read_hie (struct ghw_handler *h)
 	case ghw_hie_design:
 	case ghw_hie_eos:
 	  /* Should not be here.  */
-	  abort ();
+	  ghw_error_exit();
```

Comment:
`abort` is ok in that case (which shouldn't happen).

Reply:
Sure, changed back to abort. My thinking in changing these was that it would be more consistent with using `ghw_error_exit` everywhere.